app/views/session_timeout/_ping.js.erb

```diff
@@ -6,7 +6,12 @@ var warningEl = document.getElementById('session-timeout-cntnr');
 warningEl.insertAdjacentHTML('afterbegin', warning_info);
 
 var modal = new window.LoginGov.Modal({ el: '#session-timeout-msg' });
+var keepaliveEl = document.getElementById('session-keepalive-btn');
+
+keepaliveEl.addEventListener('click', keepalive, false);
+
 var ping_timeout;
+var countdown_interval;
```

Comment:
lets make these both snake case? `pingTimeout` `countdownInterval`?

Reply:
At some point, it might be nice to pull more / all of this code into a pack or package, to take advantage of linting there (specifically, [ESLint `camelcase`](https://eslint.org/docs/rules/camelcase)). Or, vice-versa, maybe we can lint `.js.erb` 🤔